src/webui/webapplication.cpp

```diff
@@ -402,15 +401,29 @@ void WebApplication::configure()
     m_isReverseProxySupportEnabled = pref->isWebUIReverseProxySupportEnabled();
     if (m_isReverseProxySupportEnabled)
     {
-        m_trustedReverseProxyList.clear();
-
         const QStringList proxyList = pref->getWebUITrustedReverseProxiesList().split(u';', Qt::SkipEmptyParts);
 
-        for (const QString &proxy : proxyList)
+        m_trustedReverseProxyList.clear();
+        m_trustedReverseProxyList.reserve(proxyList.size());
+
+        for (QString proxy : proxyList)
```

Comment:
Wouldn't it be a good idea to extract the logic below into `QList<Utils::Net::Subnet> parseSubnetList(const QString &str)`? You could provide Test for it.

Reply:
IP address and subnet notation aren't the same thing and I'm not sure this is eligible for its own function... maybe revisit later when there are more use of it.

Reply:
> IP address and subnet notation aren't the same thing and I'm not sure this is eligible for its own function... maybe revisit later when there are more use of it.

Ok.